front/lib/models.ts

```diff
@@ -880,6 +880,54 @@ ChatRetrievedDocument.init(
 
 ChatMessage.hasMany(ChatRetrievedDocument);
 
+export class GensTemplate extends Model<
+  InferAttributes<GensTemplate>,
+  InferCreationAttributes<GensTemplate>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+  declare workspaceId: ForeignKey<Workspace["id"]>;
+  declare instructions: string[];
+  declare name: string;
+}
+
+GensTemplate.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
```

Comment:
As per comment below, let's add a `sId` so that we can have an upsert mechanism and we don't leak database Ids when deleting